__tests__/commands/run.js

```diff
@@ -13,6 +13,7 @@ jasmine.DEFAULT_TIMEOUT_INTERVAL = 90000;
 const execCommand: $FlowFixMe = require('../../src/util/execute-lifecycle-script').execCommand;
 
 const path = require('path');
+const q = process.platform === 'win32' ? '"' : "'";
```

Comment:
Well, let's agree to add a comment above it stating this then and call it a day? :)